packages/runtime/test/transport/publicRelationshipTemplateReferences.test.ts

```diff
@@ -0,0 +1,50 @@
+import { ClientResult, PublicRelationshipTemplateReferenceClient } from "@nmshd/transport";
+import { RuntimeServiceProvider, TestRuntimeServices } from "../lib";
+import { RestClientMocker } from "../lib/RestClientMocker";
+
+const serviceProvider = new RuntimeServiceProvider();
+let runtimeServices: TestRuntimeServices;
+let clientMocker: RestClientMocker<PublicRelationshipTemplateReferenceClient>;
+
+beforeAll(async () => {
+    runtimeServices = (await serviceProvider.launch(1))[0];
+    const client = (runtimeServices.transport.publicRelationshipTemplateReferences as any).getPublicRelationshipTemplateReferencesUseCase
+        .publicRelationshipTemplateReferencesController.client as PublicRelationshipTemplateReferenceClient;
```

Comment:
we don't do any casts to access private variables. instead you can access them by using `aController["privateVariable"]`

Reply:
same for `packages/transport/test/modules/publicRelationshipTemplateReferences/PublicRelationshipTemplateReferencesController.test.ts` btw.